python/Ganga/__init__.py

```diff
@@ -727,6 +727,15 @@ def filefilter(fn):
                         'defaultProtocol': defaultMassStorageProto},
                        massStorageFileExt)
 
+sharedFileBackendPost = {'LSF': 'client', 'LCG': 'client', 'ARC': 'client', 'Dirac': 'client', 'PBS': 'client', 'Interactive': 'client', 'Local': 'client', 'CREAM': 'client'}
+
+output_config.addOption('SharedFile',
+                       {'fileExtensions': [''],
+                        'backendPostprocess': sharedFileBackendPost,
+                        'uploadOptions': {'path': '/tmp/sharedStorage', 'cp_cmd': 'cp', 'ls_cmd': 'ls', 'mkdir_cmd': 'mkdir -p'},
```

Comment:
I'm not wanting to spam all users just because we're adding an optional extra feature, that's an annoyance users will not thank us for.